File: 梦想/dream.py
# -*- coding:utf-8 -*-  
import requests
import hashlib,time

# md5 = hashlib.md5()
# md5.update('ye06021123')
# print md5.hexdigest()
#>>>9ef1969d6ed8fc9a289ef6b0e48e13b6
headers={
	'Host': 'app0.dreamlive.tv',
	'Accept': '*/*',
	'Connection': 'keep-alive',
	'app_ct': 'CN',
	'app_v': '2.0.9',
	'Accept-Encoding': 'gzip, deflate',
	'Accept-Language': 'zh-Hans-CN;q=1, en-CN;q=0.9',
	'Content-Type': 'application/x-www-form-urlencoded',
	'Content-Length': '163',
	'User-Agent': 'Dreamer/2.0.9 (iPhone; iOS 10.1.1; Scale/2.00)',
	'Connection': 'keep-alive',
	'app_type': '0',
	'Cookie': 'JSESSIONID=F9F975F8153A46BEAC2BB0F70CAE3B0E',
	'app_dev': '5C8448A1-1D70-4541-A9D3-37187C1382CA'
}
t=str(int(time.time()))
md5 = hashlib.md5()
md5.update(t)
m_check=md5.hexdigest()
print m_check
data='account=13126772351&deviceId=5C8448A1-1D70-4541-A9D3-37187C1382CA&mcheck='+m_check+'&password=9ef1969d6ed8fc9a289ef6b0e48e13b6&r='+str(int(time.time()))
# params={
# 	'account':'13126772351',
# 	'deviceId':'5C8448A1-1D70-4541-A9D3-37187C1382CA',
# 	'mcheck':'4bd4b051e5e83707ec2a4914473efeff',
# 	'password':'9ef1969d6ed8fc9a289ef6b0e48e13b6',
# 	'r':'1484561989673'
# }
# cookies={'JSESSIONID':'58E63DB365210FD77B1C1BC2E2CD3F5B'}
url='http://app0.dreamlive.tv/passport/login/AccountLoginAction.a'
requests.post(url,headers=headers,data=data)

File: 梦想/dream.js
var settings={
	operation: "GET",
	headers:{
			'Host': 'h5.weiyingonline.com',
			'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8',
			'Connection': 'keep-alive',
			'Upgrade-Insecure-Requests': '1',
			'Cookie': 'totoro_cookie=1484129147647_9300; totoro_share_click_cookie=1484129147638_1808; JSESSIONID=685BEB546B16CF9F86BA068B39BBA22D',
			'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_1_1 like Mac OS X) AppleWebKit/602.2.14 (KHTML, like Gecko) Mobile/14B100 MicroMessenger/6.5.3 NetType/WIFI Language/zh_CN',
			'Accept-Language': 'zh-cn',
			'Accept-Encoding': 'gzip, deflate',
			'Connection': 'keep-alive',
		}
}

DESKTOP_USER_AGENTS = [ "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36",
		'Mozilla/5.0 (X11; Linux x86_64; rv:29.0) Gecko/20100101 Firefox/29.0',
		'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.137 Safari/537.36',
		'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:29.0) Gecko/20100101 Firefox/29.0',
		'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.137 Safari/537.36',
		'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.75.14 (KHTML, like Gecko) Version/7.0.3 Safari/537.75.14',
		'Mozilla/5.0 (Windows NT 6.1; WOW64; rv:29.0) Gecko/20100101 Firefox/29.0',
		'Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.137 Safari/537.36',
		'Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1; SV1)',
		'Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)',
		'Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 6.1; WOW64; Trident/4.0)',
		'Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.1; WOW64; Trident/5.0)',
		'Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.1; WOW64; Trident/6.0)',
		'Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; rv:11.0) like Gecko',
		'Mozilla/5.0 (iPhone; CPU iPhone OS 10_1_1 like Mac OS X) AppleWebKit/602.2.14 (KHTML, like Gecko) Mobile/14B100 MicroMessenger/6.5.3 NetType/WIFI Language/zh_CN'

	];

//循环访问直播网页	
var i=0;
//var phantom = require('phantom');
var webPage = require('webpage');
var page = webPage.create();
var url='http://h5.weiyingonline.com/share_play/SharePlayClickAction.a?shareDetailId=1182444'
var url='http://h5.weiyingonline.com/share_play/SharePlayClickAction.a?shareDetailId=1183885'
var url='http://h5.weiyingonline.com/share_play/SharePlayClickAction.a?shareDetailId=1184579'

var page;
function iter(){
	//随机产生user-agent
	//settings.headers['User-Agent']=DESKTOP_USER_AGENTS[Math.floor(Math.random()*7)]
	i=i+1;
	console.log(i);	
//	var page=null;
	page=null;
	page = webPage.create();
	page.open(url, settings,function (s) {
		console.log(s==='success');
		while(s!=="success"){
			console.log('wait a minute');
		}

	});
	if(i>=1000){phantom.exit();}
	setTimeout(arguments.callee, 1000)
}
iter()